backend/benefit/applications/api/v1/views.py

```diff
@@ -246,6 +283,10 @@ class HandlerApplicationViewSet(BaseApplicationViewSet):
     permission_classes = [BFIsHandler]
     filterset_class = HandlerApplicationFilter
 
+    def dispatch(self, request, *args, **kwargs):
+        print("HandlerApplicationViewSet.dispatch profiling")
```

Comment:
Was this print statement left here by accident?

Reply:
yes, good catch